src/Integration/LocalServices/RuleSetInspector.cs

```diff
@@ -125,11 +175,49 @@ public RuleConflictInfo FindConflictingRules(string baseLineRuleSet, string targ
                 return null;
             }).Where(r => r != null);
 
-            var disabledRules = baseLine.Rules.Where(r => !effectiveRulesMap.ContainsKey(r.FullId));
+            var disabledRules = baselineRuleSet.Rules.Where(r => !effectiveRulesMap.ContainsKey(r.FullId));
 
             return new RuleConflictInfo(disabledRules, deprioritizedRules);
         }
 
+        /// <summary>
+        /// Attempts to fix the conflicts by updating the RuleSet Includes to what they were 
+        /// when we bound the solution to SonarQube
```

Comment:
This comment confused me. The user might have added or removed other includes since the solution was bound, but we don't know or care about such includes. Isn't this method trying to ensure the server ruleset is included with the expected include action?